http_service/pulse/consumers.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Library with Pulse factory that can generate Pulse consumers.
+
+Import the ConsumerFactory and generate the connection & consumer you need.
+
+```python
+    # Righ now the factory only has hg_pushes to generate a connection & consumer
+    connection, consumer = ConsumerFactory.hg_pushes(user, password, _on_message)
+    with connection as conn:  # noqa
+        consumer.run()
+```
+"""
+from kombu import Connection, Exchange, Queue
+from kombu.mixins import ConsumerMixin
+
+_CONNECTION_URL = "amqp://{}:{}@pulse.mozilla.org:5671/?ssl=1"
+
+
+def _generate_hg_pushes_queue(user):
```

Comment:
Nit: you could just inline this function in the `hg_pushes` function below, it's just a line after all (split into multiple because of its length, but still not much code)